src/main/java/se/liu/ida/hefquin/engine/queryplan/physical/impl/PhysicalOpBindJoin.java

```diff
@@ -58,4 +59,17 @@ public void visit(final PhysicalPlanVisitor visitor) {
         visitor.visit(this);
     }
 
+    @Override
+    public String toString() {
+        final StringBuilder builder = new StringBuilder();
+
+        if ( lop instanceof LogicalOpTPAdd)  {
+            builder.append( "> bindJoin");
+            return ( (LogicalOpTPAdd) lop ).toString();
+        }
+        else {
+            throw new IllegalArgumentException("Unsupported type of operator: " + lop.getClass().getName() );
+        }
```

Comment:
Instead of having these if-else part with this exception, why don't you simply use `lop.toString()`, independent of the type of `lop`.

Reply:
The same issue repeats in the other physical operators as well.